lib/kontentClient.ts

```diff
@@ -76,10 +76,10 @@ export const getHomepage = (usePreview: boolean) =>
       waitForLoadingNewContent: usePreview
     })
     .depthParameter(10)
-    .toPromise()
+    .toAllPromise()
```

Comment:
This is unnecessary because we expect only one item to match the criteria and only take the first item in the following `then`.

Reply:
adjusted